M2/Macaulay2/packages/Complexes/ChainComplex.m2

```diff
@@ -592,6 +622,11 @@ poincareN Complex := C -> (
     f
     )
 
+rank Complex := ZZ => C -> (
+    (lo, hi) := concentration C;
+    sum for i from lo to hi list (-1)^i * rank C_i
```

Comment:
Out of curiosity, is this different than the Euler characteristic of the complex?

Reply:
That might be true for vector spaces, but in general it is similar.  We only added this for backward compatibility, as there was a function in Core doing this, and code using that.